src/checkers/inference/InferrableAnnotatedTypeFactory.java

```diff
@@ -0,0 +1,32 @@
+package checkers.inference;
+
+import org.checkerframework.framework.type.treeannotator.TreeAnnotator;
+
+/**
+ * Interface for the annotatedTypeFactory that wish to let
+ * InferenceAnnotatedTypeFactory use the TreeAnnotator from current type system.
```

Comment:
Make this comment more general about what the interface is for. The TreeAnnotator is only relevant for the one method in the interface. In the future there might be more.